src/physics/em/detail/SBEnergyDistribution.hh

```diff
@@ -0,0 +1,135 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file SBEnergyDistribution.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "physics/base/Units.hh"
+#include "physics/grid/TwodSubgridCalculator.hh"
+#include "random/distributions/UniformRealDistribution.hh"
+#include "SeltzerBerger.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Sample exiting photon energy from Bremsstrahlung.
+ *
+ * The SB energy distribution uses tabulated Seltzer-Berger differential cross
+ * section data, which stores cumulative probabilities as a function of
```

Comment:
These aren't cumulative probabilities, are they?

Reply:
Quite correct. I am revising the documentation now -- hopefully will remove the squirrel infestation as well as fixing other technical errors like this.